CHANGELOG/CHANGELOG-2.3.md

```diff
@@ -4,10 +4,18 @@ Changelog for the ZDM Proxy, new PRs should update the `unreleased` section.
 
 When cutting a new release, update the `unreleased` heading to the tag being generated and date, like `## vX.Y.Z - YYYY-MM-DD` and create a new placeholder section for `unreleased` entries.
 
+## Unreleased
+
+* [#143](https://github.com/datastax/zdm-proxy/issues/143): Update ANTLR dependency
```

Comment:
if we don't have anything else we are working on you can already put this as 2.4.0 and do the release after the merge